src/components/TableRadioTeams/components/SkeletonTable/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {Children} from "react";
+import {Skeleton} from "@/components/ui/skeleton";
+import {SkeletonTableProps} from "./types";
+import clsx from "clsx";
```

Comment:
Ordenado de los imports

```suggestion
import React, {Children} from "react";
import clsx from "clsx";

import {Skeleton} from "@/components/ui/skeleton";

import {SkeletonTableProps} from "./types";
```